### DISCUSSION TITLE
How can budibase print screen to PDF?

### DISCUSSION THREAD

**Discussion by LTCTHITA** [Budibase/budibase#12023] [Help] [answered]
Hey, I want to save a PDF file from screen on budibase. What do I have to do?

**Comment by zedascouves** [comment, marked as answer]
The option I used was to create a page with everything I needed (text, images, formats, etc) and select the browser "print" functionality. Then, I can select "print to PDF". You may need to fiddle with the "scale" options on the print dialog pop-up.

**Reply by LTCTHITA** [reply]
Hey, @zedascouves 
I don't know how to save it as a PDF file. Can you please tell me how?

Thank you,

**Reply by zedascouves** [reply]
It varies a little from browser to browser, but a Google search does wonders: https://www.wikihow.com/Save-a-Web-Page-as-a-PDF-in-Google-Chrome.
